schedule/rotation/store.go

```diff
@@ -259,6 +266,42 @@ func (s *Store) FindMany(ctx context.Context, ids []string) ([]Rotation, error)
 	return result, nil
 }
 
+func (s *Store) FindManyByUserID(ctx context.Context, tx *sql.Tx, userID string) (rotationIDs []string, err error) {
```

Comment:
This should return `[]Rotation` to match things like `FindMany` also to reduce round-trips.

Could be done in a number of ways:
```sql
select * from rotations
where id = any(select rotation_id from rotation_participants where user_id = $1)
```

Reply:
would it be okay to return a `[]Rotation` slice, where only the `ID` is set on each el?